### PR TITLE
Add code of conduct

### LINKED ISSUE #22: Add code of conduct
<a href="https://github.com/samanpwbb"></img></a> **Issue by [samanpwbb](https://github.com/samanpwbb)**
_Tuesday Jun 02, 2020 at 17:22 GMT_
_Originally opened as https://github.com/jdalt/twin-cities-aid-distribution-locations/issues/31_

----

https://github.com/codeforamerica/codeofconduct is a good starting point

### PR DESCRIPTION
**Pull request by cwood821**
### What
Adds a code of conduct to the repository.

### Why
Encourage and establish expectations from contributors. Resolves #22. 

Note: removed QA checklist as this is purely a markdown addition and does not change map behavior.

### CODE REVIEW

**Comment by cwood821** [comment]
I adapted (in some cases very lightly or without alteration) the Code for America code of conduct, as suggested in #22. I read several other code of conducts and added a few points beyond CFA's that seemed relevant and also updated the civic tech points to be less general and more focused on local community, which seemed in the spirit of the project. 

Please let me know any and all feedback!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+The Twin Cities Aid Map comunity welcomes participation from everyone.
```
Comment (jdalt) [inline CODE_OF_CONDUCT.md]:
```suggestion
The Twin Cities Mutual Aid Map community welcomes participation from everyone. 
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+The Twin Cities Aid Map community welcomes participation from everyone.
```
Comment (jdalt) [inline CODE_OF_CONDUCT.md:3]:
```suggestion
The Twin Cities Mutual Aid Map community welcomes participation from everyone. 
```

Should probably match the organization and the website url.

**Comment by jdalt** [comment]
I really like how simple and concise this is.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+The Twin Cities Mutual Aid Map community welcomes participation from everyone. 
+
+## Expectations 
+
+Our community expects that participants:
```
Comment (kmarekspartz) [inline CODE_OF_CONDUCT.md:7]: What happens when members don't meet these expectations?

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+* Create a community where members are safe, respected, friendly, and free to fully express their identities
+* Provide an environment where people are free from discrimination and harassment
+* Take responsibility for their words and actions
+* Value everyone’s ideas, skills, and contributions
+* Encourage participation, questions, and discussion
+* Don’t assume everyone has the same context 
+* Find ways for people to productively contribute their time and skills (technical or not)
```
Comment (grebbesen) [inline CODE_OF_CONDUCT.md:15]: What different ways can we welcome, encourage and acknowledge all contributions without having two buckets? I read `technical` here as `code` (and find it easy to read a hierarchy in that relationship) but one could argue that data analysis or any other range of skills are technical. OTC has been working to elevate outreach, process/system mapping, data gathering and analysis and other "non-technical" activities. I know this is a bullet from the CfA CoC (https://brigade.codeforamerica.org/about/code-of-conduct) but welcome ideas.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (grebbesen) [reply CODE_OF_CONDUCT.md:7]: From the CfA CoC https://brigade.codeforamerica.org/about/code-of-conduct

> If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, including warning the offender or expulsion from Code for America network activities, events, and digital forums.

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (grebbesen) [reply CODE_OF_CONDUCT.md:15]: ?
> Value everyone’s ideas and find ways for people to productively contribute their time and skills

Comment (alexives) [reply CODE_OF_CONDUCT.md:15]: I like that proposal!

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (cwood821) [reply CODE_OF_CONDUCT.md:7]: This is a really good point! Thanks for calling that out. I'll add the CfA language.

File: CODE_OF_CONDUCT.md (hunk 4 above)
Comment (cwood821) [reply CODE_OF_CONDUCT.md:15]: @grebbesen That's a great point. I agree on the hierarchy implication. I'll remove the qualification for now. I do wonder if the context of Github overly implies code and/or if qualification is necessary at all (I'm genuinely not sure). Happy to update with any ideas from others!

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (cwood821) [reply CODE_OF_CONDUCT.md]: Good catch! Thanks.

**Comment by cwood821** [comment]
@kmarekspartz @grebbesen thanks for the feedback! This should be ready for a second review based on your comments.

**Comment by kmarekspartz** [comment]
That brings up a good point: does or should this apply to the data curation side (FB group, Google sheet) or to visitors of the Website or to visitors of the physical sites they discover on the Website?

**Comment by alexives** [comment]
One suggestion, maybe we should link to it in the readme like:
```diff
diff --git a/README.md b/README.md
index fecbd8f..26edeb3 100644
--- a/README.md
+++ b/README.md
@@ -34,12 +34,13 @@ This started as a very lightweight, single page html file, and we've tried very
 
 ### Setup
 
-1. Install node at the version specified in the [`.node_version`](.node_version) file. If you use a version manager like nodenv or nvm, this should be detected automatically.
-2. Install dependencies with npm
+1. Review the [Code of Conduct](CODE_OF_CONDUCT.md)
+2. Install node at the version specified in the [`.node_version`](.node_version) file. If you use a version manager like nodenv or nvm, this should be detected automatically.
+3. Install dependencies with npm
     ```
     npm install
     ```
-3. Start building!
+4. Start building!
 ```

**Comment by experimatt** [comment]
This looks great, we can add a link to the COC from the Readme separately after merging this in.

**Pull request merged**